Extensions/Settings/Dnn.PersonaBar.SqlConsole/admin/personaBar/scripts/sort.js

```diff
@@ -24,7 +24,7 @@ define(['jquery', 'knockout'], function ($, ko) {
 
     var changeSort = function (e) {
         var $element = $(e.target);
-        var viewModel = ko.dataFor($element.parent()[0]);
+        var viewModel = ko.dataFor($element.parent().parent().parent()[0]);
```

Comment:
This is an awkward construct and potentially brittle in case of changes to element hierarchy. Is there no other way to get that view model?

Reply:
If we plan to redesign this module and migrate to a modern framework, I would totally agree and we can of course make it better. I just fixed a bug avoiding any kind of breaking change. I don't think it's worth to refactor a Knockout based module since it won't be anything much better than now.

Reply:
Sure, that's why I marked this as comment and not "changes requested". 

Yes, changes to existing DOM structured won't do much good. I'm more asking if this is the best approach for current codebase.